BraveWallet/Crypto/Stores/SwapTokenStore.swift

```diff
@@ -572,6 +571,7 @@ public class SwapTokenStore: ObservableObject {
             return
           }
           self.state = .error(Strings.Wallet.unknownError)
+          self.clearAllAmount()
```

Comment:
so it will reset if there is an unknown error? i actually dont know what is the expected reset behaviour. But I'm ok with this.

Reply:
Yes, I checked for other usage of `clearAllAmount()` / field resets and we seem to only reset when we hit an unknown error. For this issue specifically, we were getting the insufficient liquidity error which I don't think should reset the value. Similar for insufficient balance error. The 'swap' button updates for both of those errors so I don't think we need to reset value fields.